ddl/index.go

```diff
@@ -779,14 +789,23 @@ func (w *worker) waitTaskResults(workers []*addIndexWorker, taskCnt int, totalAd
 	return nextHandle, addedCount, errors.Trace(firstErr)
 }
 
+var gofailMockAddindexErrOnceGuard bool
+
 // handleReorgTasks sends tasks to workers, and waits for all the running workers to return results,
 // there are taskCnt running workers.
-func (w *worker) handleReorgTasks(startTime time.Time, startHandle int64, reorgInfo *reorgInfo, totalAddedCount *int64, workers []*addIndexWorker, batchTasks []*reorgIndexTask) error {
+func (w *worker) handleReorgTasks(reorgInfo *reorgInfo, totalAddedCount *int64, workers []*addIndexWorker, batchTasks []*reorgIndexTask) error {
 	for i, task := range batchTasks {
+		// gofail: var mockAddIndexErr bool
+		// if mockAddIndexErr && !gofailMockAddindexErrOnceGuard {
```

Comment:
`gofailMockAddindexErrOnceGuard` is too long.

Reply:
To avoid name conflicts.